src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -17,6 +17,7 @@ $font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, 'Noto Sans', sans-s
 
 $badge-font-size: 1rem;
 $form-feedback-font-size: 100%;
+$popover-max-width: 350px;
```

Comment:
```suggestion
.popover-content.popover-body div {
  max-width: 350px;
  text-overflow: ellipsis;
  overflow: hidden;
}
```
![screenshot](https://user-images.githubusercontent.com/1897962/69353942-a2353f80-0c7f-11ea-92cf-f9a4355df8a7.png)

Reply:
how about simply breaking the line? I don't like the ideia of hiding some of the information that you typed
![image](https://user-images.githubusercontent.com/399326/69535715-0c91fc80-0f6c-11ea-80ee-1755ac9b3477.png)